yt_dlp/extractor/acast.py

```diff
@@ -43,14 +43,14 @@ class ACastIE(ACastBaseIE):
     _VALID_URL = r'''(?x:
                     https?://
                         (?:
-                            (?:(?:embed|www)\.)?acast\.com/|
+                            (?:(?:embed|www|shows)\.)?acast\.com/|
                             play\.acast\.com/s/
                         )
-                        (?P<channel>[^/]+)/(?P<id>[^/#?"]+)
+                        (?P<channel>[^/]+)/(?:episodes/)?(?P<id>[^/#?"]+)
```

Comment:
I know this is old code, but I'm always gonna fix it when I see it
```suggestion
                        (?P<channel>[^/?#]+)/(?:episodes/)?(?P<id>[^/#?"]+)
```

Reply:
If you mean you'll adapt this when merging: Please do.

I have a hard time keeping up with all the "best known methods" around yt-dlp's coding. `traverse_obj` still makes my mind spin. ;-)